src/components/NewWorkspaceModal.js

```diff
@@ -118,12 +118,13 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
   )(() => Promise.all([
     Ajax(signal).Billing.listProjects()
       .then(_.filter({ status: 'Ready' }))
+      .then(_.filter(project => isBillingProjectApplicable(project)))
       .then(projects => {
         setBillingProjects(projects)
         setNamespace(_.some({ projectName: namespace }, projects) ? namespace : undefined)
       }),
     Ajax(signal).Groups.list().then(setAllGroups),
-    !!cloneWorkspace && Ajax(signal).Workspaces.workspace(namespace, cloneWorkspace.workspace.name).checkBucketLocation(cloneWorkspace.workspace.googleProject, cloneWorkspace.workspace.bucketName)
+    !!cloneWorkspace && !cloneWorkspace.azureContext && Ajax(signal).Workspaces.workspace(namespace, cloneWorkspace.workspace.name).checkBucketLocation(cloneWorkspace.workspace.googleProject, cloneWorkspace.workspace.bucketName)
```

Comment:
The checkBucketLocation call will error for Azure workspaces. We are going to disable the Clone option until it is fully supported for Azure workspaces, so I could choose not to add this check… but while testing this PR I was distracted/confused by the error popping up.